base/src/main/java/org/mozilla/jss/pkcs12/PFX.java

```diff
@@ -211,11 +215,14 @@ public PFX( AuthenticatedSafes authSafes ) {
      */
     public void computeMacData(Password password,
             byte[] salt, int iterationCount)
-        throws NotInitializedException, DigestException,
+        throws NotInitializedException, DigestException,NoSuchAlgorithmException,
         TokenException, CharConversionException
     {
+
+        //Make this alg the default mac alg.
+        AlgorithmIdentifier algID = new AlgorithmIdentifier(DigestAlgorithm.SHA256.toOID());
```

Comment:
I understand that you are changing the default mac alg, but is there not a need to allow caller of computeMacData() to specify algID for better future support?